frontend-client/app/_components/NavLink.tsx

```diff
@@ -0,0 +1,24 @@
+'use client'
+
+import Link from 'next/link'
+import { usePathname } from 'next/navigation'
+
+interface NavLinkProps {
+  href: string
+  text: string
+}
+
+export default function NavLink({ href, text }: NavLinkProps) {
+  const pathname = usePathname()
+
+  return (
+    <Link
+      href={href}
+      className={`text-lg hover:opacity-60 ${
+        pathname === href ? 'text-primary' : ''
```

Comment:
<img width="766" alt="Screenshot 2023-12-29 at 1 48 39 PM" src="https://github.com/skkuding/codedang/assets/50468628/fd912413-4124-476a-9ef7-215cf0f92140">

지금처럼 구현하면 하위 주소가 있을시에 적용이 안됩니다! 따라서 포함이 되는지를 확인하는게 좋을 것 같아요!

```suggestion
        pathname.includes(href) ? 'text-primary' : ''
```

Reply:
includes로 하면 뒤에 포함되어있어도 적용되지 않나요?
예를 들어서 /group/notice라는 링크도 notice에 적용이 될 거 같은데,
`includes()` 대신 `startsWith()` 쓰는 거 어떨까요?

Reply:
앗 그 경우를 생각 못했네요! 그게 좋을 것 같습니다!

Reply:
그렇네요. 오늘안으로 확인하고 적용하겠습니다!